client/src/components/SiteTile.js

```diff
@@ -0,0 +1,32 @@
+import React from "react"
+
+const SiteTile = ({ name, address, description, environment, minimumAge }) => {
+    return(
+        <>
+            <h2>
+                {name}
+            </h2>
```

Comment:
can all be on the same line
```
<h2>{name}</h2>
```
unless content spans much farther (this applies to all HTML elements in this component)